<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>displayplot</class>
 <widget class="QMainWindow" name="displayplot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="ChartWork" name="m_chart" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="m_channel">
     <property name="title">
      <string>导联</string>
     </property>
     <addaction name="separator"/>
     <addaction name="m_channel_1"/>
     <addaction name="m_channel_2"/>
     <addaction name="m_channel_3"/>
     <addaction name="m_channel_4"/>
     <addaction name="m_channel_5"/>
     <addaction name="m_channel_6"/>
     <addaction name="m_channel_7"/>
     <addaction name="m_channel_8"/>
     <addaction name="m_channel_all"/>
    </widget>
    <widget class="QMenu" name="m_sampling">
     <property name="title">
      <string>采样率</string>
     </property>
     <addaction name="m_sampling_250"/>
     <addaction name="m_sampling_500"/>
     <addaction name="m_sampling_1000"/>
    </widget>
    <addaction name="m_channel"/>
    <addaction name="m_sampling"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="m_channel_1">
   <property name="text">
    <string>导联：1</string>
   </property>
  </action>
  <action name="m_channel_2">
   <property name="text">
    <string>导联：2</string>
   </property>
  </action>
  <action name="m_channel_3">
   <property name="text">
    <string>导联：3</string>
   </property>
  </action>
  <action name="m_channel_4">
   <property name="text">
    <string>导联：4</string>
   </property>
  </action>
  <action name="m_channel_5">
   <property name="text">
    <string>导联：5</string>
   </property>
  </action>
  <action name="m_channel_6">
   <property name="text">
    <string>导联：6</string>
   </property>
  </action>
  <action name="m_channel_7">
   <property name="text">
    <string>导联：7</string>
   </property>
  </action>
  <action name="m_channel_8">
   <property name="text">
    <string>导联：8</string>
   </property>
  </action>
  <action name="m_channel_all">
   <property name="text">
    <string>导联：全部</string>
   </property>
  </action>
  <action name="m_sampling_250">
   <property name="text">
    <string>250</string>
   </property>
  </action>
  <action name="m_sampling_500">
   <property name="text">
    <string>500</string>
   </property>
  </action>
  <action name="m_sampling_1000">
   <property name="text">
    <string>1000</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChartWork</class>
   <extends>QWidget</extends>
   <header>chartwork.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
